static/app/components/events/breadcrumbs/utils.tsx

```diff
@@ -63,10 +63,11 @@ const Color = styled('span')<{colorConfig: ColorConfig}>`
  * As of writing this, it just grabs a few, but in the future it may collapse,
  * or manipulate them in some way for a better summary.
  */
-export function getSummaryBreadcrumbs(crumbs: EnhancedCrumb[], sort: BreadcrumbSort) {
-  const breadcrumbs = [...crumbs];
-  const sortedCrumbs =
-    sort === BreadcrumbSort.OLDEST ? breadcrumbs : breadcrumbs.reverse();
+export function getSummaryBreadcrumbs(
+  crumbs: readonly EnhancedCrumb[],
+  sort: BreadcrumbSort
+) {
+  const sortedCrumbs = sort === BreadcrumbSort.OLDEST ? crumbs : crumbs.toReversed();
```

Comment:
not really necessary but reduces arrays by one and couldn't resist using toReversed